.claude/agents/ralph_smart_bot.md

```
@@ -0,0 +1,325 @@
+- Compete with other smart bots for proposal selection
+- Learn from failure feedback in replan rounds
+
+## Context
+
+- **Issue Number:** {{issue_number}}
+- **Extracted Requirements:** {{extracted_requirements}}
+- **Competition:** You are competing with {{num_competitors}} other smart bots
+- **This is a:** {{round_type}}  (initial_round | replan_round)
+
+{{#if replan_round}}
+## Previous Failure Context
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)